blinky.md

```diff
@@ -12,11 +12,7 @@ Blinking some lights is the "Hello World" of hardware. Let's make those LEDs bui
 
 In your command line, make a folder for your Tessel code, then initialize a Tessel project in that folder by running each of the following commands in the terminal:
 
-`mkdir tessel-code`
-
-`cd tessel-code`
-
-`t2 init`
+`mkdir tessel-code; cd tessel-code; t2 init`
```

Comment:
that's what I want to do, but it doesn't work. It just looks like in your code block, fences visible..
see https://github.com/tessel/t2-start/issues/46

I wanted to make it copy/pasteable as a group, but still look red like the other bash commands